src/virtual_dom/node/el.rs

```diff
@@ -9,6 +9,26 @@ use crate::browser::{
 };
 use std::borrow::Cow;
 
+// ------ ElKey ------
+
+#[allow(clippy::module_name_repetitions)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ElKey(String);
+
+#[allow(clippy::module_name_repetitions)]
+/// Attach given `key` to the `El`.
+///
+/// ## Warning
+///
+/// WIP
+///  - it doesn't patch elements with the different keys, BUT it replace the old element with the new one.
+///  - `key` restriction (have to implements `ToString`) will be probably relaxed in the future.
+pub fn el_key(key: &impl ToString) -> ElKey {
```

Comment:
Should it be `Display` instead? Citing docs:
> This trait is automatically implemented for any type which implements the Display trait. As such, ToString shouldn't be implemented directly: Display should be implemented instead, and you get the ToString implementation for free.

Reply:
Yeah, I know this advice. I don't implement `ToString`, I'm just using `impl` instead of generics here.
I think - if I understand that advice correctly - that `ToString` is basically something like a subset of `Display`. So I chose to write `&impl ToString` because it sounds to me a little bit more relaxed than `Display`. And I don't want to "display" the key anywhere, I just want to convert it to `String` so it feels also more descriptive to me.